src/main/scala/ApplicationRoutes.scala

```diff
@@ -0,0 +1,19 @@
+import ApplicationEncodersDecoders._
+import cats.effect.Async
+import org.http4s.HttpRoutes
+import org.http4s.dsl.Http4sDsl
+import cats.implicits._
+
+object ApplicationRoutes {
+
+  def contributorsWrapper[F[_]: Async](githubClient : GitHubClient[F])  = {
+    val dsl = new Http4sDsl[F] {}
+    import dsl._
+    HttpRoutes.of[F]{
```

Comment:
Although it's fine, I'd not be worried for exposing this yet

Reply:
Could you please elaborate on this comment? What do you mean by `exposing this yet`?

Reply:
Yes, I mean we can build the API in a following task.